packages/components/pie-link/src/index.ts

```diff
@@ -20,6 +18,10 @@ const componentSelector = 'pie-link';
  */
 
 export class PieLink extends LitElement implements LinkProps {
+    @property()
+    @validPropertyValues(componentSelector, tags, 'a')
```

Comment:
I think using `isButton` could be simpler and remove the need for this validation